.github/workflows/main.yml

```diff
@@ -79,6 +78,7 @@ jobs:
     name: Publish release
     permissions:
       contents: write
+      id-token: write # Required for requesting the OIDC JWT for cloud access token
```

Comment:
This is used for publishing to npm. For AWS, we. use the `publish-registry` job above.

Reply:
Moved to `publish-registry`